src/app/app.component.ts

```diff
@@ -12,9 +12,7 @@ export class AppComponent implements OnDestroy {
 
   mobileQuery: MediaQueryList;
 
-  menu = ['find-journey', 'create journey', 'list journey'];
-
-  fillerNav = this.menu;
+  fillerNav = ['find-journey', 'create journey', 'list journey'];
 
   fillerContent = Array.from({length: 50}, () =>
```

Comment:
You can change the content to put for example "DaKar, travel as you imagine", or "DaKar, compose your journey of your dreams", or "DaKar, just take your valise"....